dipy/conftest.py

```diff
@@ -0,0 +1,27 @@
+"""pytest initialization."""
```

Comment:
Does this ever get used by anything? I couldn't find an import of this module anywhere.

Reply:
`Pytest` look for many setup files implicitly. `conftest.py` is one of them. more info [here](https://docs.pytest.org/en/2.7.3/plugins.html)

Reply:
Huh. Interesting.